data/quarg/quarg news.txt

```diff
@@ -506,3 +506,4 @@ phrase "quarg poems on species"
 		"Make strongest neighbor / See through the guise / Let none forget / The curse of the wise"
 		"With skin of stone / Might, meek, guile / Abilities to hone / And wars to pile"
 		"Twice-blessed pilgrims / Myths of the past / Masses preserve / The royal caste"
+		"Adrift by the clouds / Under watchful eyes / Begin now a journey / To the greatest skies"
```

Comment:
It works as-is, of course. But I can't help but feel like you're missing an opportunity:
```suggestion
		"Adrift by the clouds / Under watchful eyes / Begin now a journey / To the endless skies"
```

Reply:
Personally I'm not a fan of name-dropping ES's game title; the one instance where I think Endless Sky would work is as the name of an Archon.